GeneralsMD/Code/GameEngine/Source/GameClient/MessageStream/SelectionXlat.cpp

```diff
@@ -1099,10 +1100,15 @@ GameMessageDisposition SelectionTranslator::translateGameMessage(const GameMessa
 					m_lastGroupSelTime = now;
 				}
 
+				Bool performSelection = TRUE;
+
 				// check for double-press to jump view
 				if ( now - m_lastGroupSelTime < 20 && group == m_lastGroupSelGroup )
 				{
 					DEBUG_LOG(("META: DOUBLETAP select team %d",group));
+					// TheSuperHackers @tweak Stubbjax 26/05/2025 Perform selection on double-press
```

Comment:
Is this a tweak or a fix or a bugfix? I was under the impression this was fixing a user facing bug.